src/java/org/apache/cassandra/concurrent/Stage.java

```diff
@@ -46,7 +46,7 @@
     MUTATION          (true,  "MutationStage",         "request",  DatabaseDescriptor::getConcurrentWriters,        DatabaseDescriptor::setConcurrentWriters,        Stage::multiThreadedLowSignalStage),
     COUNTER_MUTATION  (true,  "CounterMutationStage",  "request",  DatabaseDescriptor::getConcurrentCounterWriters, DatabaseDescriptor::setConcurrentCounterWriters, Stage::multiThreadedLowSignalStage),
     VIEW_MUTATION     (true,  "ViewMutationStage",     "request",  DatabaseDescriptor::getConcurrentViewWriters,    DatabaseDescriptor::setConcurrentViewWriters,    Stage::multiThreadedLowSignalStage),
-    ACCORD            (true,  "AccordStage",           "request",  DatabaseDescriptor::getConcurrentAccordOps,      DatabaseDescriptor::setConcurrentAccordOps,      Stage::multiThreadedLowSignalStage),
+    ACCORD            (true,  "AccordStage",           "request",  () -> 0,                                         null,                                            Stage::immediateExecutor),
```

Comment:
Isn't it confusing to call this the `AccordStage`? Shouldn't we just pass the IMMEDIATE stage?

Reply:
pushed the following change; removed `ACCORD` and all usages now uses `IMMEDIATE`